src/repository/user.local.pg.repository.ts

```diff
@@ -0,0 +1,151 @@
+import Database from './database.interface';
+import UserRemote from '../model/entity/user.remote.type';
+import UserLocal from '../model/entity/user.local.type';
+import ListUser from '../model/request/list.user.type';
+
+const { PreparedStatement } = require('pg-promise');
+
```

Comment:
This is not respecting the 80 char limit

Reply:
There seems to be conflict of rules with my eslint configuration. For some reason it really likes to break my manual indentation and favor a one-line solution. I would need more time to debug this problem. This is not intended.